docs/4devs.md

```diff
@@ -30,3 +30,17 @@ For _very_ rough statistics, open devTools -> Network and look at the line that
 ## Subscriptions and queries
 
 For the performance of the app, it is important that we limit the amount of queries and subscriptions that alchemy creates when loading a page.
+
+## Coding conventions
+
+- As an ongoing process of adopting language localization, any new static text should be added to a translation file and used via `i18next` internationalization-framework.
+Translation files location is under `src/assets/locales/`. After adding your text just import `i18next` and use the translation function:
+
+  ```
+  import i18next from "i18next";
+
+  i18next.t("YOUR TEXT FROM THE TRANSLATION FILE")}
```

Comment:
Should it be clear that 
"YOUR TEXT FROM THE TRANSLATION FILE" should be the _key_?

Reply:
Like "The key of the record you added to the translation file"

Reply:
Check it now.